java/app-encryption/src/test/java/com/godaddy/asherah/appencryption/SessionFactoryTest.java

```diff
@@ -85,6 +86,8 @@ void testConstructor() {
 
   @Test
   void testSessionCacheSetupAndClose() {
+    when(metastoreMock.getRegionSuffix()).thenReturn("");
```

Comment:
can move this mock setup to the `@BeforeEach` annotation. can help to get rid of some redundant code.